packages/core/admin/admin/src/pages/HomePage/tests/index.test.js

```diff
@@ -0,0 +1,1283 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Router } from 'react-router-dom';
+import { createMemoryHistory } from 'history';
+import { IntlProvider } from 'react-intl';
+import { ThemeProvider, lightTheme } from '@strapi/parts';
+import HomePage from '../index';
+
+jest.mock('../../../hooks', () => ({
+  useModels: jest.fn(() => ({ isLoading: false, collectionTypes: [], singleTypes: [] })),
```

Comment:
Here you can write to different tests, one for the view when there is no collectionTypes and no singletypes and one for when there is at lest one of them?